app/views/reviews/_review_editor.html.erb

```diff
@@ -172,14 +172,16 @@
           <span data-bind="text: averageGrade"></span>
         </h3>
       <!-- /ko -->
-      
+
       <!-- Next page button -->
-      <!-- ko if: nextPage || $root.finishable() -->
-        <button class="btn" data-bind="click: $root.showNextPage">
+        <button class="btn" data-bind="enable: nextPage || $root.finishable(), click: $root.showNextPage">
```

Comment:
@tsauvine I updated this pull request. You can review it and give me the signal to merge.